contracts/factories/SRC20Registry.sol

```diff
@@ -7,15 +7,22 @@ import "../interfaces/ISRC20Registry.sol";
 
 
 /**
- * @dev SRC20 registry contains addresses of every created 
- * SRC20 token. Registered factories can put addresses of
+ * @dev SRC20 registry contains the address of every created 
+ * SRC20 token. Registered factories can add addresses of
  * new tokens, public can query tokens.
  */
 contract SRC20Registry is ISRC20Registry, Manager {
     using Roles for Roles.Role;
 
-    Roles.Role private _factories;
+    event FactoryAdded(address account);
+    event FactoryRemoved(address account);
+    event SRC20Registered(address token, address tokenOwner);
+    event SRC20Removed(address token);
+    event MinterAdded(address minter);
+    event MinterRemoved(address minter); // @TODO move to interface
```

Comment:
Clear todo by moving event declarations to interface, and some of these are already declared in `ISRC20Registry`.

Reply:
Done